src/vms/pvm/index.ts

```diff
@@ -1,7 +1,9 @@
-export * from './builder';
+export * from './etna-builder';
 export * from './models';
 export * from './api';
 export * from './txs/fee';
 
-// Exposed Etna builder functions under `e` namespace
+/**
+ * @deprecated Exposed Etna builder functions under `e` namespace
```

Comment:
Let's update the deprecated message to something useful to point the consumer to do. ie:

"PVM builder functions aliased under "e" are deprecated. Please use the builder functions on the root `pvm` export. Ex: `pvm.e.newBaseTx` -> `pvm.newBaseTx`."